lxfspacy.py

```diff
@@ -0,0 +1,37 @@
+import spacy
+
+language_dict = {
+  "pt": {
+    "speed": "pt_core_news_sm",
+    "accuracy": "pt_core_news_lg"
+  },
+  "en": {
+   "speed": "en_core_web_sm",
+   "accuracy": "en_core_web_trf"
+  },
+  "ru": {
+    "speed": "ru_core_news_sm",
+    "accuracy": "ru_core_news_lg"
+  }
+}
+ 
+def nlp(lang, opt):
+  return spacy.load(language_dict[lang][opt])
+
+#function getLexemes
```

Comment:
1. If you add a comment, add a space between the `#` and the comment (just stylling)
2. This comment doesn't describes what your code does. It's just the name of the function. Try to explain what it is doing.